tests/test_server.py

```diff
@@ -0,0 +1,155 @@
+"""
+Inventory API Service Test Suite
+
+Test cases can be run with the following:
+  nosetests -v --with-spec --spec-color
+  coverage report -m
+"""
+
+from flask_api import status
+from mock import MagicMock, patch
+from models import ProductInformation, DataValidationError, db
+import json
+import logging
+import os
+import server
+import unittest
+
+######################################################################
+#  Fixed Global Variables
+######################################################################
+DATABASE_URI = os.getenv('DATABASE_URI', 'sqlite:///db/test.db')
+# Data table entry names
+PROD_ID = 'prod_id'
```

Comment:
Maybe we should create a module specifically for storing constants so we don't create multiple versions of the same constants in different modules, which is error-prone for code maintenance in the future. I'll create a issue for that.